home/pi/piScreen/piScreenUtils.py

```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/python3
 import logging, logging.handlers, __main__, os
+from enum import Enum
 
-class Paths():
+
+# TODO: Replace this line with `class Paths(StrEnum)' and add `from enum import StrEnum' to the
```

Comment:
I added this comment to remind future devs of removing this hack, so they don't forget to use `StrEnum` at some point in the future, although I have no idea when Python 3.11 will be widely adopted. By doing that, we can ensure that this hack won't persist forever.